msteams-platform/sbs-teams-app-test-tool.yml

```diff
@@ -0,0 +1,351 @@
+
+    1. Copy and save the **Deployment name** for further use.
+    
+        :::image type="content" source="~/assets/images/test-tool/copy-deployment.png" lightbox="~/assets/images/test-tool/copy-deployment.png" alt-text="Screenshot shows the deployment name for Azure open AI deployment.":::
+
+    ## Update Azure Open AI key and endpoints
+
+    1. Open your project in Visual Studio Code.
+
+    1. Under **EXPLORER**, go to **env** > **.env.testtool.user** file.
+
+    1. Enter your **SECRET_AZURE_OPENAI_API_KEY** and **SECRET_AZURE_OPENAI_ENDPOINT**.
+
+        ```text
+        ...
+        SECRET_AZURE_OPENAI_API_KEY=<azure-openai-api-key>
+        SECRET_AZURE_OPENAI_ENDPOINT=<azure-openai-endpoint>
+        ```
+
+    1. Go to **src** > **app.js** file.
+
+    1. Comment the `OpenAI` code and uncomment the `Azure OpenAI` code.
+
+    1. Enter your Azure Open AI deployment name in `azureDefaultDeployment`.
+
+        ```JavaScript
+        ...
+        // Use OpenAI
+        // apiKey: config.openAIKey,
+        // defaultModel: "gpt-3.5-turbo",
+
+        azureApiKey: config.azureOpenAIKey,
+        azureDefaultDeployment: "gpt-35-turbo",
+        azureEndpoint: config.azureOpenAIEndpoint,
+        ...
+        ```
+
+    ## Debug and run your AI chat bot app
+
+    1. From the left pane, select **Run and Debug** (`Ctrl+Shift+D`) and select **Debug in Test Tool (Preview)** from the dropdown list.
+
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/debug/select-debug-in-test-tool.png" lightbox="~/assets/images/teams-toolkit-v2/debug/select-debug-in-test-tool.png" alt-text="Screenshot shows the option to select debug in test tool.":::
+ 
+    1. The Test Tool opens your AI chat bot in a webpage.
+
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/debug/test-tool.png" lightbox="~/assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the bot open in Test Tool.":::
+
+- title: Activity triggers
+  durationInMinutes: 1
+  content: |
+
+    You can mock an activity in Test Tool using activity triggers. There are two types of activity triggers:
+
+    1. [Predefined activity triggers](#predefined-activity-triggers)
+    1. [Custom activity triggers](#custom-activity-triggers)
+
+    ### Predefined activity triggers
+
+    Test Tool provides predefined activity triggers to test the functionalities of your bot.
+
+      | Category | Activity | Handler |
+      | --- | --- | --- |
+      | Trigger Installation Update Activity | Install bot <br><br><br> Uninstall bot | `onInstallationUpdate` <br> `onInstallationUpdateAdded` <br><br> `onInstallationUpdate` <br> `onInstallationUpdateRemove`|
+      | Trigger Conversation Update Activity | Add user <br><br> Add bot <br><br> Add channel | `onMembersAdded`<br><br> `onTeamsMembersAddedEvent` <br><br> `onTeamsChannelRenamedEvent` |
```

Comment:
There're two handlers for 'Add user':
1. `onMembersAdded`
2. `onTeamsMembersAddedEvent`

And same two handlers (with different payload) for 'Add bot':
1. `onMembersAdded`
2. `onTeamsMembersAddedEvent`

The handler for 'Add channel' should be: `onTeamsChannelAddedEvent`

Reply:
Updated the doc as suggested